app/src/main/res/layout/scan_result_dialog.xml

```diff
@@ -62,13 +65,13 @@
         android:layout_marginTop="6dp"
         android:paddingHorizontal="10dp"
         android:paddingVertical="6dp"
-        android:background="@drawable/qr_result_background"
         android:maxHeight="200dp"
         android:autoLink="all"
         android:scrollbars="vertical"
         android:fadeScrollbars="false"
         android:overScrollMode="always"
         android:textIsSelectable="true"
-        android:layout_width="match_parent" />
-    
+        android:layout_width="match_parent"
+        android:textColor="@color/qr_results"/>
```

Comment:
This view inherits colors for the text color, so no custom color is needed. If we want a more prominent color than the default, we can use `?android:attr/textColorPrimary`, which is the most prominent text color and a [night-qualified color resource](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#themes-styles).

```suggestion
        android:textColor="?android:attr/textColorPrimary"/>
```